### ISSUE TITLE
Dynamic Group detail membership tab pagination is broken

### ISSUE THREAD

**Issue by jathanism** [nautobot/nautobot#2219]
### Environment
* Python version: *
* Nautobot version: 1.4.0

### Steps to Reproduce
1. Navigate to a Dynamic Group detail view in the web UI that has more than 25 members
2. Click next page 
3.

### Expected Behavior

View should stay on the "members" tab.

### Observed Behavior

It returns you to the "main" tab. It also does not persist the page size setting.

**Comment by jathanism** [comment]
The issue is that when you click to paginate the Members tab, the `tab=members` URL fragment doesn't persist (and neither does the one for `page_size=`). Seems safe to assume fixing one may fix both.

**Comment by jathanism** [comment]
After thinking I had arrived at a fix, I determined that I had not. But I can assert correct behavior like so:

1. Navigate to a Dynamic Group detail view in the web UI that has more than 25 members
2. Refresh the page
3. Observe the URL fragments are correct
4. For that session it works as intended until you navigate away from the Members tab

There's a state persistence issue here. Perhaps in our JS history handling code as it relates to nav tabs.
